CMakeLists.txt

```diff
@@ -67,6 +67,13 @@ endif()
 # NOTE: Policy changes has to happen before adding any subprojects
 cmake_policy(SET CMP0091 NEW)
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+OPTION(PRODUCTION_BUILD_OPT "Define Production or Engineering build" OFF)
```

Comment:
I did check in the official PRODUCTION_BUILD flags yesterday. 
Don't use the flags above, use the ones used in the whole source code.

Reply:
Done